istioctl/pkg/auth/testdata/validator/unused-role.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: "rbac.istio.io/v1alpha1"
+kind: ServiceRole
+metadata:
+  name: unused-role
+  namespace: default
+spec:
+  rules:
+    - services: ["productpage.svc.cluster.local"]
```

Comment:
NIT: it should be productpage.default.svc.cluster.local

Reply:
I thought we removed this namespace because the namespace is already in metadata so we will know where this service lives. https://istio.io/docs/reference/config/authorization/istio.rbac.v1alpha1/ also does not include namespace in services. Do we still want to follow k8s dns standard to include namespace?

Reply:
We need to fix the doc as well.

Reply:
Done.